README.md

```diff
@@ -9,29 +9,29 @@
     <img src="https://badge.fury.io/js/eslint-config-mimic.svg" alt="NPM">
   </a>
   <a href="https://discord.mimic.fi">
-    <img src="https://img.shields.io/discourse/status?server=https%3A%2F%2Fmeta.discourse.org" alt="Discord">
+    <img src="https://img.shields.io/discord/747115528198332456" alt="Discord">
```

Comment:
This one seems invalid, see:

https://img.shields.io/discord/747115528198332456